polyfactory/value_generators/complex_types.py

```diff
@@ -136,6 +137,16 @@ def handle_collection_type_coverage(
         return container.union(handle_collection_type_coverage(field_meta, set, factory, build_context=build_context))
 
     if issubclass(container_type, tuple):
+        if field_meta.children[-1].annotation == Ellipsis:
```

Comment:
Ideally this and other methods should respect `__randomize_collection_length__`. This is a bigger change so have left this change out of this PR